### PR TITLE
remove duplicated THIRD_PARTY.TXT, WARNING.TXT

### PR DESCRIPTION
**Pull request by chipitsine**
Changes proposed in this pull request:
 - remove duplicated text file

Your great patch is much appreciated. We are considering to apply your patch into the SoftEther VPN main tree.

SoftEther VPN Patch Acceptance Policy:
http://www.softether.org/5-download/src/9.patch

You have two options which are described on the above policy.
Could you please choose either option 1 or 2, and specify it clearly on the reply?

- one

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -70,9 +70,9 @@ STATEMENT FOR WARNING AND DISCLAIMER.
 THE FOLLOWING GPLV2 CONDITIONS APPLY ON ALL SOFTETHER VPN PROGRAMS WHICH ARE
 DEVELOPED BY SOFTETHER VPN PROJECT.
 
-READ AND UNDERSTAND THE 'WARNING.TXT' FILE BEFORE USING THIS SOFTWARE.
+READ AND UNDERSTAND THE 'src/WARNING.TXT' FILE BEFORE USING THIS SOFTWARE.
```
[the remark this answers is not in the document]
Comment (chipitsine) [reply LICENSE]: it is LICENSE, not LICENSE.md
if it is acceptable to convert license to markdown, it would be great

**Comment by davidebeatrici** [comment]
I would remove the files in `src` and keep the ones in the repository root.

**Comment by chipitsine** [comment]
files in src are used in "centos/SPECS/softethervpn.spec"

File: LICENSE (hunk 1 above)
[the remark this answers is not in the document]
Comment (moatazelmasry2) [reply LICENSE]: Ok I understand the problem. I prefer to do the renaming separated in its own PR. In this case, please rename all TXT files that should have been .md, for example:
- AUTHORS.TXT
- LICENSE
- THIRD_PARTY.TXT
- WARNING.TXT

Also in my openion, all these text files (except README.md) should not have all capitalized letters in their names

[the remark this answers is not in the document]
Comment (chipitsine) [reply LICENSE]: those files are also used in "centos/SPECS/softethervpn.spec"

I doubt they are really needed in rpm (maybe due to legal terms?), also, I'm not sure md is good there


so, I just removed duplicates. if you wish, please feel free to open any other PR on your own

**Comment by chipitsine** [comment]
@moatazelmasry2 - I was sure it is 'src/*'

I can check after May 10, I'm on vacation now (with limited access to servers).

**Comment by chipitsine** [comment]
it turned out that spec was broken (surprise! by myself) in https://github.com/SoftEtherVPN/SoftEtherVPN/pull/395


what I suggest is

1) merge this PR (i.e. 502)
2) merge https://github.com/SoftEtherVPN/SoftEtherVPN/pull/302
3) setup CI for rpm build (I think about Fedora COPR)

otherwise spec will be broken almost all the time (and work by chance from time to time)

**Comment by moatazelmasry2** [comment]
Ok cool. Will try to get it done today. Still need to fix some merge conflicts and change a version number in in PR #302 before doing that. Are you aware of anyone using this spec files??

**Comment by chipitsine** [comment]
well, spec is broken (I check both SoftEtherVPN and SoftetherVPN_Stable), nobody complains, so 100% nobody is using it.

we'll need to decide what to do with redhat packaging (it might be either "it's up to distributive maintainers" or "here's OpenSuse Build Service or Fedora COPR with prebuilt RPMs). we can do it later, it is ok to have master branch broken for a while (it's already broken  from spec point of view).

I'm interested in maintaining COPR packages. Hope to find some time to set it up soon

**Comment by moatazelmasry2** [comment]
How about setting up separate repos for Fedora, Debian etc.. as they don't necessarily belong to the core SE Codebase?

**Comment by chipitsine** [comment]
repo itself means not too much.

maintaining packages is responsibility. it might be done either inside distributive (Debian is known for that) or inside package (i.e. providing repos with actual prebuilt packages like Launchpad, Fedora COPR or similar clouds)

when you agreed on responcibility, you can choose either way, i.e. inside or outside repo. I'd prefer to have specs inside repo (it allows simple push triggers for CI)

**Comment by chipitsine** [comment]
it was rebased

**Comment by chipitsine** [comment]
oh, come on. it was rebased just as you asked...

**Comment by moatazelmasry2** [comment]
Yes sorry. My mistake. I just fixed it

**Pull request merged**